src/Helper/MyParcelCollection.php

```diff
@@ -300,6 +305,18 @@ public function createConcepts(): self
         return $this;
     }
 
+    /**
+     * Label prepare wil be active from x number of orders
+     *
+     * @param int $numberOfOrders
+     *
+     * @return bool
+     */
+    public function getLimitLabelPrepare(int $numberOfOrders): bool
```

Comment:
Bij het terugkrijgen van een bool, moet je denk ik toch geen getter gebruiken. Maar use, has, can, is, must.
useLabelPrepare of zoiets

Reply:
Die parameter is niet nodig. Je kan die waar de ook halen uit `$this->count()`